src/games/strategy/triplea/settings/SettingInputComponentFactory.java

```diff
@@ -139,6 +144,7 @@ private static JPanel createRadioButtonPanel(
   private static <T extends HasDefaults> SettingInputComponent<T> build(
       JPanel componentPanel,
       LabelDescription labelDescription,
+      final Optional<ValueRange> valueRange,
```

Comment:
Optional params are a smell. Looks like can be potentially be fixed by representing `Optional.empty()` with a `ValueRange.ANY` object that will print out to an empty string.